go_covert_lib/build_watch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
This is a build script that will automatically run instances of the server at ports 8080 and 8081. It also runs a watch script which auto-rebuilds the client whenever changes are made. This script is not required to run, but it is there for your convenience if you would like.